notebooks/getting_started.ipynb

```diff
@@ -184,19 +286,78 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
     "del labeler.lfs[-1]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## LF backlog\n",
+    "\n",
+    "- ~~ST elevation (common sign of heart attack)~~\n",
+    "    - ~~STEMI|ST elevation~~\n",
+    "- obstruction of heart vessels (occurs in 15-20% of people with heart disease)\n",
+    "    - xx \n",
+    "    - %|percent \n",
+    "    - blockage|obstruction|occlusion|narrowed \n",
+    "    - of|in|around \n",
+    "    - coronary arteries(or arteries) | Left coronary artery | LCA | Left anterior descending artery | LAD | Left circumflex artery | Posterior descending artery | Right coronary artery | RCA | Right marginal artery | Posterior descending artery | PDA\n",
+    "- swelling/edema (commmon symptom of heart failure)\n",
+    "    - swelling|edema|puffiness\n",
+    "    - in \n",
+    "    - left | right | l | r (optional)\n",
+    "    - ankle(s), leg(s), feet | foot\n",
+    "- angina (common symptom of coronary artery disease)\n",
+    "    - stable|unstable|variant (optional)\n",
+    "    - angina|chest pain|angina pectoris\n",
+    "- abnormal diagnostic test results\n",
+    "    - abnormal|concerning \n",
+    "    - ECG|echo|echocardiogram\n",
+    "- ~~atherosclerosis~~\n",
+    "    - ~~atherosclerosis|arteriosclerosis|atherosclerotic|arterial sclerosis|artherosclerosis|atherosclerotic disease~~\n",
+    "- ~~heart attack~~\n",
+    "    - ~~myocardial infarcation|MI|ischemic heart disease|cardiac arrest|coronary infacrtion|asystole|cardiopulmonary arrest|coronary thrombosis|heart arrest|heart attack|heart stoppage~~\n",
+    "- ~~heart failure~~\n",
+    "    - ~~congestive heart failure|decomensated heart failure|CHF|left-side heart failure|right-sided heart failure~~\n",
+    "- correlated procedures: \n",
+    "    - coronary|cardiac cath|catheter|catheterization\n",
+    "    - coronary|cardiac stent|stenting|angioplasty\n",
+    "    - Percutaneous coronary intervention|PCI\n",
+    "- correlated drugs (get the individual drug names from SNOMED for the following SNOMED categories, and exact/synonym match for these in the text. At least 1 hit should be a POSITIVE: \n",
+    "    - Cardiovascular Agents (all of them)\n",
+    "    - Hematologic agent\n",
+    "    - Thrombolytic\n",
+    "    - Anticoagulant\n",
+    "- correlated diseases (get the individual drug names from SNOMED for the following SNOMED categories, and exact/synonym match for these in the text. At least 1 hit should be a POSITIVE: \n",
+    "    - Anything with ICD10CM chapter 9 (get all of the code names/synonyms and exact match)\n",
+    "- correlated cardiac markers - regex, similar to ejection fraction, to search for cardiac markers with abnormal ranges:\n",
+    "    - see this: https://en.wikipedia.org/wiki/Cardiac_marker\n",
```

Comment:
Let's just collect these in a list somewhere else? A google doc perhaps. I can do this

Reply:
@karthikraja95 Do you want to try your hand at some of these?

Or even evaluate some of the LFs Nick added to the notebook?

Let me know if you also run into issues with the preprocessing step.

Reply:
For sure. I will check it out